ansible/vtestbed.yaml

```diff
@@ -351,3 +351,20 @@
   inv_name: veos_vtb
   auto_recover: 'False'
   comment: Tests virtual switch vm
+
+- conf-name: vms-kvm-smartswitch
```

Comment:
where is the topology update between NPU to DPU VMs?

Reply:
I only created VM for NPU and DPU in this PR.
We will have another PR to cover the connection between NPU to DPU VMs.